helper_functions.py

```diff
@@ -809,3 +819,22 @@ def reference_lines(x_range, y_start, slopes, line_type='log'):
             shift = y_start/y_range[0,ss]
             y_range[:,ss] = y_range[:,ss]*shift
     return(y_range)
+
+def lowess_mean(ds,win_size,lowess_delta):
```

Comment:
Looks like `model4D_calcQOIs()` already uses lowess--is the implementation the same? Can you update that function to call your new function?